app/controllers/idv/getting_started_controller.rb

```diff
@@ -0,0 +1,87 @@
+module Idv
+  class GettingStartedController < ApplicationController
+    include IdvStepConcern
+    include StepUtilitiesConcern
+
+    before_action :confirm_agreement_needed
+
+    def show
+      analytics.idv_doc_auth_getting_started_visited(**analytics_arguments)
+
+      # Register both Welcome and Agreement steps in DocAuthLog
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('welcome', :view, true)
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('agreement', :view, true)
+
+      @sp_name = decorated_session.sp_name || t('doc_auth.info.no_sp_name')
+      @title = t(
+        'doc_auth.headings.getting_started',
+        sp_name: decorated_session.sp_name || t('doc_auth.info.no_sp_name').downcase,
+      )
+
+      render :show, locals: { flow_session: flow_session }
+    end
+
+    def update
+      flow_session[:skip_upload_step] = true unless FeatureManagement.idv_allow_hybrid_flow?
```

Comment:
what would happen if we wrote an explicit `false` here? like if we did:
```suggestion
      flow_session[:skip_upload_step] = !FeatureManagement.idv_allow_hybrid_flow?
```

Reply:
I'm assuming it's written this way for a reason (copied from Welcome step), AND this should be localized to a HybridHandoff before_action. Added [LG-10402](https://cm-jira.usa.gov/browse/LG-10402) to deal with :skip_upload_step in general. I'd prefer to leave it as-is in the current PR.

Reply:
sounds good!